src/AssetsTransferApi.ts

```diff
@@ -329,6 +375,10 @@ export class AssetsTransferApi {
 			return Direction.SystemToRelay;
 		}
 
+		if (isSystemParachain && SYSTEM_PARACHAINS_IDS.includes(destChainId)) {
```

Comment:
I saw a comment by @joepetrowski  that all `para_ids` less than 2000 can be considered a system parachain, so that might help us from having to have an array with all the ID's to identify if its a system para. 

I think in this PR we can keep it as is, but its good to keep that in mind. I'll write an issue to keep track of this for us.

Reply:
exact same thought came to my mind when I saw that. Agree we should keep it in mind so an issue is great